scripts/docker/Dockerfile.gcc

```diff
@@ -1,4 +1,22 @@
-FROM gcc:5.3.0
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+    build-essential \
+    wget \
+    git \
+    bc \
+    ccache \
+    && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN echo "deb http://dk.archive.ubuntu.com/ubuntu/ xenial main" >> /etc/apt/sources.list && \
+    echo "deb http://dk.archive.ubuntu.com/ubuntu/ xenial universe" >> /etc/apt/sources.list && \
+    apt-get update && apt-get install -y g++-5 && \
+    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-5 5 && \
+    update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-5 5 && \
```

Comment:
Can you confirm that this yields GCC 5.3 and not version 5.4?

Reply:
yes, I have checked